nncf/quantization/algorithms/smooth_quant/torch_fx_backend.py

```diff
@@ -42,7 +42,7 @@
 class FXSQMultiply(torch.nn.Module):
     def __init__(self, scale: torch.Tensor):
         super().__init__()
-        self._scale_value = scale
+        self._scale_value = torch.nn.Parameter(scale)
```

Comment:
I would suggest to Buffer instead of Parameter.

Reply:
Done